docs/modules/ROOT/nav.adoc

```diff
@@ -18,6 +18,7 @@
 .Reference
 // Configuration options/spec files/any other reference docs
 * xref:phone-homes.adoc[Phone Homes]
+* xref:jvm-parameters.adoc[JVM Parameters]
```

Comment:
Why this page is listed under `Reference`? I think it should be listed with other features at `Get Started`, right?

Reply:
@hasancelik. If you decide to move the page, could you change the title to `Configuring JVM Parameters`?

Reply:
@dzeromski-hazelcast I think we should move it. Could you change the title also? :)